Clarify effects of suspending users and server (#1218)
mastodon/documentation: content/en/admin/moderation.md

Once the data has been deleted, whether that is after the 30 day period, or if an admin has force deleted it, the account can still be un-suspended. However, the account will have no data (statuses, profile information, avatar or header image) associated with it.

> For remote accounts, suspending will remove their existing follow relationships with local accounts. Those relationships are not restored in case the remote account is un-supended, even within the 30-day time window.

Reviewer: This is actually a bit subtler, local local followers still follow the remote user, but remote followers stop following local users.

Author: Ok, If that's the case it should definitely be documented because it's even more confusing.

Reviewer: What about the following?

```suggestion
For remote accounts, suspending will make them unfollow any local account. Those relationships are not restored in case the remote account is un-supended, even within the 30-day time window.
```

Author: Yes, that works and it’s  ore precise. Thanks
